docs/events_handling.md

```diff
@@ -0,0 +1,102 @@
+
+# Events handling
+
+## Subscribed events
+
+Precaution needs to subscribe to a number of events
+to satisfy different use cases.
+
+The events are separated into two groups `pull requests
+events` and `check suites/runs events`.
+
+### Pull requests events
+
+Official GitHub documentation: [PullRequest event](https://developer.github.com/v3/activity/events/types/#pullrequestevent)
+
+#### `pull_request.opened`
+
+- __occurs__: when a new pull request is created*
+
+- __triggers__: start of the Precaution scan*
+
+#### `pull_request.reopened`
+
+- __occurs__: when a pull request is reopened from a closed state*
+
+- __triggers__: start of a new Precaution scan*
+
+#### `pull_request.synchronize`
+
+- __occurs__: when you add a commit to already opened pull request*
+
+- __triggers__: start of a new Precaution scan*
+
+For more information look at pull request number [#113](https://github.com/vmware/precaution/pull/113)
+
+
+### Check run and check suite events
```

Comment:
As above, didn't we agree to use theGitHub event names for the section titles?

Reply:
Yes, my mistake.